docs: update version/doctor output (#1956)
iterative/dvc.org: content/docs/command-reference/version.md

| `Supports`                                  | Types of [remote storage](/doc/command-reference/remote/add#supported-storage-types) supported by the current DVC setup (their required dependencies are installed)  |
| `Cache types`                               | [Types of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)     |
| `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                                |

> | `Caches`                                    | [Types of caches](/doc/user-guide/managing-external-data) (e.g. local, s3, ssh, etc.) configured in the repo                                                         |

Reviewer: ```suggestion
| `Caches`                                    | Cache [location types](/doc/user-guide/managing-external-data) configured in the repo (e.g. local, SSH, S3, etc.)                                                         |
```
>Needs formatting.

> | `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                           |

Author: ```suggestion
| `Remotes`                                   | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                             |
```

> | `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                            |

Author: ```suggestion
| `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                           |
```

> | `Remotes`                                   | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                             |

Reviewer: ```suggestion
| `Remotes`                                   | Remote [location types](/doc/command-reference/remote/add#supported-storage-types) configured in the repo (e.g. SSH, S3, Google Drive, etc.)                                                             |
```
> Also needs formatting.

```dvc
$ dvc version

> DVC version: 1.10.0 (pip)

Author: just to not confuse people with the older DVC version

Reviewer: No strong opinion but there's another 3 instances of `1.7` in this doc.

Author: Updated, thanks!

---------------------------------
Platform: Python 3.8.3 on Linux-5.4.0-47-generic-x86_64-with-debian-buster-sid
Supports: gs, hdfs, http, https, s3
Cache types: hardlink, symlink
Cache directory: ext4 on /dev/sda1
Caches: local

> Remotes: https

Author: ```suggestion
Remotes: https, ssh
```
